server/static/static.go

```diff
@@ -83,8 +83,13 @@ func handleRootPaths(env environment.Env, rootPaths []string, template *template
 
 func serveIndexTemplate(env environment.Env, tpl *template.Template, version string, w http.ResponseWriter) {
 	issuers := make([]string, 0)
+	ssoEnabled := false
 	for _, provider := range env.GetConfigurator().GetAuthOauthProviders() {
-		issuers = append(issuers, provider.IssuerURL)
+		if provider.Slug == "" {
+			issuers = append(issuers, provider.IssuerURL)
+		} else {
+			ssoEnabled = true
+		}
```

Comment:
nit: add a comment like:

// Assemble a slice of the supported issuers. Omit "private" issuers, which have a slug,
// and set ssoEnabled = true if any private issuers are present in the config.

Reply:
Done